sms_sponsorship/models/sms_child_request.py

```diff
@@ -363,8 +363,8 @@ def sms_notification_unfinished_cron(self):
             self.message_post(
                 body=_("{} partner(s) have ongoing SMS Sponsorship").format(
                     nb_sms_requests),
-                subject=_("{} SMS Sponsorship are ongoing").format(
-                    nb_sms_requests),
+                subject=_("{} SMS Sponsorship {} ongoing").format(
+                    nb_sms_requests, 'is' if nb_sms_requests <= 1 else 'are'),
```

Comment:
is and are should be wrapped in the translation function.
```suggestion
                    nb_sms_requests, _('is') if nb_sms_requests <= 1 else _('are')),
```

Reply:
Done.